app/javascript/angular/code/controllers/_sessions_list_ctrl.js

```diff
@@ -39,15 +34,15 @@ export const SessionsListCtrl = (
     // when refactoring to a radio button this should always be false
     return (
       !params.get("selectedSessionIds", []).includes(sessionId) &&
-      sessions.hasSelectedSessions()
+      sessionsUtils.isSessionSelected()
     );
   };
 
   $scope.$watch(
     "params.get('map')",
     ({ hasChangedProgrammatically }) => {
       console.log("watch - params.get('map')");
-      if (sessions.hasSelectedSessions()) return;
```

Comment:
The fact that this was working before is MAGIC! `hasSelectedSessions ` was calling a chain of at least 5 other methods and at the end is a function that does 
```_(self.sessions || []).detect(function(session) {
      return session.id === id;
    });
```
But when a page with selected session is first loaded (or refreshed) the `self.sessions` is empty so `sessions.hasSelectedSessions()` returned false event though session was selected 😮